draft-ietf-acme-acme.md

```diff
@@ -1985,6 +1985,8 @@ domain by verifying that the resource was provisioned as expected.
 5. Verify that key authorization provided by the HTTP server matches the token
    for this challenge and the client's account key.
 
+The server MAY follow redirects when dereferencing the URI.
```

Comment:
How about `SHOULD`?  It seems like having a clear expectation here is in the interest of interoperability.